src/main/scala/org/hammerlab/guacamole/commands/jointcaller/kryo/Registrar.scala

```diff
@@ -0,0 +1,38 @@
+package org.hammerlab.guacamole.commands.jointcaller.kryo
+
+import com.esotericsoftware.kryo.Kryo
+import org.apache.spark.serializer.KryoRegistrator
+import org.hammerlab.guacamole.commands.jointcaller.Parameters.SomaticGenotypePolicy
+import org.hammerlab.guacamole.commands.jointcaller.annotation.{InsufficientNormal, MultiSampleAnnotations, SingleSampleAnnotations, StrandBias}
+import org.hammerlab.guacamole.commands.jointcaller.evidence.{MultiSampleMultiAlleleEvidence, MultiSampleSingleAlleleEvidence, NormalDNASingleSampleSingleAlleleEvidence, TumorDNASingleSampleSingleAlleleEvidence, TumorRNASingleSampleSingleAlleleEvidence}
+import org.hammerlab.guacamole.commands.jointcaller.{AlleleAtLocus, Input, InputCollection, Parameters}
+
+class Registrar extends KryoRegistrator {
+  override def registerClasses(kryo: Kryo): Unit = {
+    kryo.register(classOf[NormalDNASingleSampleSingleAlleleEvidence])
+    kryo.register(classOf[TumorDNASingleSampleSingleAlleleEvidence])
+
+    // These are needed for MultiSampleSingleAlleleEvidence; perTumorDnaSampleTopMixtures and probably other fields.
+    kryo.register(Class.forName("scala.collection.immutable.MapLike$$anon$2"))  // MapLike.MappedValues?
+    kryo.register(Class.forName("org.hammerlab.guacamole.commands.jointcaller.evidence.MultiSampleSingleAlleleEvidence$$anonfun$16"))
```

Comment:
Yea I'm not even sure what anonymous inner-classes or methods these are referring to; I discussed this a little bit when I added them: https://github.com/hammerlab/guacamole/pull/441#issuecomment-211189236. 

I'm not sure what kinds of changes would cause these magic strings to change, but they would break tests so we shouldn't have to worry about them changing without us noticing.

Other options are:
* `@transient` or `def` some of the `val`s in `MultiSampleSingleAlleleEvidence`.
* write a dedicated `Serializer` for `MultiSampleSingleAlleleEvidence` and/or classes that are currently getting ser'd as part of its closure.